rpcs3/Emu/RSX/GL/GLHelpers.cpp

```diff
@@ -22,8 +22,8 @@ namespace gl
 		case rsx::primitive_type::quads: return GL_TRIANGLES;
 		case rsx::primitive_type::quad_strip: return GL_TRIANGLE_STRIP;
 		case rsx::primitive_type::polygon: return GL_TRIANGLES;
+		default: fmt::throw_exception("unknow primitive type" HERE);
```

Comment:
unknowN

Reply:
Put all throws on their own line. breakpoints are the only way to obtain backtraces to identify the culprit

Reply:
Good catch, thanks @OverlordCW .

Should be addressed @kd-11 .